tests/neg/cos.hs

```diff
@@ -0,0 +1,5 @@
+module CosNegTest where
```

Comment:
Thanks @renanroberto for rebasing develop.

Please, use the same name for the modules and the files.

Also, all of the new tests modules need to be added in tests.cabal for CI and our test scripts to pick them.

Lastly, the negative tests need to tell to LH that they are expected to fail with directives like:
```
{-@ LIQUID "--expect-error-containing=AbsNegTest.hs:6:1" @-}
```

See #2158

Reply:
Thanks for the help, @facundominguez! Applied the changes suggested